src/gui/dialogs/addon/manager.cpp

```diff
@@ -183,6 +184,10 @@ const std::vector<std::pair<ADDON_TYPE, std::string>> addon_manager::type_filter
 	{ADDON_UNKNOWN,        N_("addons_of_type^Unknown")},
 };
 
+std::vector<std::pair<int, std::string>> addon_manager::language_filter_types_{
+	{0,	"en_GB"},
```

Comment:
Remove this. This should not be a static member. See comment with declaration.

Reply:
:+1: 
I initially hardcoded it, so in the language filter method it would be easier to default to English, if there are no translations listed in the addon itself. In the new commit I now remember the position of en_GB with an integer